python/graphstorm/trainer/glem_np_trainer.py

```diff
@@ -45,6 +46,7 @@ def __init__(self, model, rank, topk_model_to_save=1):
         assert isinstance(model, GSgnnNodeModelInterface) and isinstance(model, GLEM), \
                 "The input model is not a GLEM node model. Please implement GLEM."
         self.early_stop = False # used when early stop is True
+        self.num_pretrain_epochs = model.num_pretrain_epochs
```

Comment:
i wonder why `num_pretrain_epochs` is not an argument of `GLEMNodePredictionTrainer`?